LoggingKata.Test/TacoParserTests.cs

```diff
@@ -8,22 +8,32 @@ public class TacoParserTests
         [Fact]
         public void ShouldDoSomething()
         {
-            // TODO: Complete Something, if anything
+            //
         }
 
         [Theory]
-        [InlineData("Example")]
+        [InlineData("0, 0, test")]
         public void ShouldParse(string str)
         {
-            // TODO: Complete Should Parse
+            //Arrange
+            TacoParser tacoParser = new TacoParser();
+            //Act
+            ITrackable result = tacoParser.Parse(str);
+            //Assert
+            Assert.NotNull(result);
         }
 
         [Theory]
         [InlineData(null)]
         [InlineData("")]

```

Comment:
Consider adding some more InlineData to test different cases that should fail, such as "test, test, test", "0, 0", "0, test, test", "100Test, 20, Test", etc.